src/ui/Popup.js

```diff
@@ -0,0 +1,81 @@
+import { Object3D } from 'three/core/Object3D';
+import { Texture } from 'three/textures/Texture';
+import { SpriteMaterial } from 'three/materials/SpriteMaterial';
+import { Sprite } from 'three/objects/Sprite';
+
+export default class Popup extends Object3D {
+  constructor(text) {
+    super();
+
+    this.sprite = createSprite(text);
+    this.initPosY = -2;
+    this.sprite.position.set(0, this.initPosY, 0);
+
+    this.posY = this.initPosY;
+    this.upPosY = -1;
+    this.timer = 0;
+    this.selected = false;
+    this.isUp = false;
+    this.isMovingUp = true;
+    this.timer = 0;
+    this.timeLimit = 80;
+    this.add(this.sprite);
+  }
+
+  update(selector) {
+    // TODO: Rewrite this animation using tweens.
```

Comment:
This works for now but in the future we should use tweens for these kinds of animations. I added a todo comment to remind us.